pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.pkgproj

```diff
@@ -40,12 +40,152 @@
         <Exclude>Compile</Exclude>
       </Dependency>
 
+      <!-- Only need the placeholder in the netcoreapp1.0 folder -->
       <File Include="$(PlaceholderFile)" >
         <SkipPackageFileCheck>true</SkipPackageFileCheck>
         <TargetPath>lib/netcoreapp1.0</TargetPath>
       </File>
     </ItemGroup>
   </Target>
 
+  <!--
+    Manually include the netcoreapp1.0 dependencies.
+    If any versions change in the release/1.0.0 branch they should
```

Comment:
@mellinoe I would love to kill the regex hack, but we need the dependencies to still get updated by the auto-update PR's which currently in core-setup we haven't added the necessary support to update them in the pkgproj (@dagood can confirm that assumption). Even if we do that we don't gain much in this scenario as we need the netcoreapp1.0 and netcoreapp1.1 to have different versions.